### DISCUSSION TITLE
json permission policy

### DISCUSSION THREAD

**Discussion by emma-fillinhgham** [iterate-ch/cyberduck#14952] [Q&A]
I have created a permission using a json file that works in other cloud access providers, like S3 Browser. These permissions do not appear to work with Cyberduck. I would like to be able to create unique folder permissions using Cyberduck, is this possible?

**Comment by tatze96** [comment]
From within Cyberduck, you can only access and adjust the ACLs. Policies are configured and applied on the server side and can't be managed within Cyberduck.

In summary, json permission policies aren't supported by Cyberduck.
